css/style.css

```diff
@@ -6,3 +6,136 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+:root{
+    --brown: #a52a2a;
+    --white: #fff;
+    --darkblue: #00001a;
+    --gray: #808080; 
+}
+
+ body {
+    margin: 0;
+    text-align: center;
+ }
+
+ /* Header */
+
+ .header{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    background: url("../img/R.jpeg");
+    padding: 0px 0px 205rem 0px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    filter: opacity(0.9);
+    /* filter: blur(1px); */
+}
+
+ .title{
+    font-size: 50px;
+    padding: 10px;
+    margin: 2px;
+    color: var(--darkblue);
+    /* background-color: var(--white); */
+ 
+}
+
+ /* Navigation */
+
+.nav_list{
+    display: flex;
+    justify-content: flex-end;
+}
+
+.nav_item{
+    list-style: none;
+}
+
+.nav_link{    
+    padding: 0px 20px;
+    font-size: 20px;
+    font-weight: bold;
+    text-decoration: none;
+    color: var(--darkblue);
+}
+
+.nav_link:hover{
+    color: var(--brown) ;
+}
+
+/* Main */
+
+.container{
+    background-color: var(--white);
+    font-size: 20px;
+    margin: 0px 200px;
+    border-radius: 2ch;
+}
+
+.container p{
+    text-align: justify;
+}
+
+.container:first-child{
+    font-style: italic;
+}
+
+.position_1{
```

Comment:
So the class name here is a little general and it is not completely clear to what you are referring to. I would suggest giving the article a name and using that as the class name. Likewise, potentially we could achieve positioning the articles without using absolute which would help with when you need a responsive page down the line. Is there anything else you could think of that might achieve the same?